interface/main/calendar/find_appt_popup.php

```diff
@@ -378,7 +378,7 @@ function setappt(year,mon,mday,hours,minutes) {
 
 <?php if (!empty($slots)) : ?>
```

Comment:
Brought this script into main codebase (PR is at https://github.com/openemr/openemr/pull/1129)